### ISSUE TITLE
Windows Filename Incompatibility

### ISSUE THREAD

**Issue by dtroberts** [saxix/drf-api-checker#4]
* DRF API Checker version: 0.9.0
* Django version: 3.1.1
* Python version: 3.7.4
* Operating System: Windows 10

### Description

Hello,

I'm trying to get things working in Windows 10, but filename incompatibility is getting in the way. The problem is that serialized objects incorporate the body into the filename, and some commonly serialized characters (`:,"` for example) are invalid in Windows filenames. 

### What I Did

Traceback:
```
Traceback (most recent call last):
  File "C:\Users\test\example\sample\tests.py", line 52, in test_payload_create
    }, name='create')
  File "C:\Python37\lib\site-packages\drf_api_checker\unittest.py", line 135, in assertPOST
    expect_errors=expect_errors, name=name)
  File "C:\Python37\lib\site-packages\drf_api_checker\unittest.py", line 99, in _assertCALL
    expect_errors=expect_errors, name=name, data=data)
  File "C:\Python37\lib\site-packages\drf_api_checker\recorder.py", line 246, in assertCALL
    _write(self.filename, serialize_response(response))
  File "C:\Python37\lib\site-packages\drf_api_checker\utils.py", line 26, in _write
    open(dest, 'wb').write(content)
OSError: [Errno 22] Invalid argument: "C:\\Users\\test\\example\\sample\\_api_checker\\api.tests\\ExampleTestCase\\create/post/{'data': {'type': 'Payload', 'attributes': {'name': 'test_payload_a', 'stage_id': 2, 'value': 'TEST_PAYLOAD_VALUE'}}}.response.json"
```

### Possible Solutions

I don't see the need to serialize the entire data object into the filename, so I think the cleanest way to fix this would be to slugify the object instead.  Alternatively, if this is required behavior, we can just sanitize all known offending characters in Windows file paths in fs.clean_url(). Other utilities are built to do this (see: https://pypi.org/project/pathvalidate/) but it's simple enough to do by manually if you want to avoid adding the requirement.

**Comment by saxix** [comment]
this is weird, filenames should not have this pattern. Are you using pytest o standard Django TestCase?

**Comment by dtroberts** [comment]
This was written using the standard Django TestCase class. 

What is weird about it to you, the nesting? Other than that, it matches what you have in the tests folder in this repository. I'm even unable to clone this repository on my Windows machine for the same reason: 

```
Cloning into 'C:\Users\test\git\drf-api-checker'...
remote: Enumerating objects: 446, done.        
remote: Counting objects: 100% (446/446), done.        
remote: Compressing objects: 100% (216/216), done.        
remote: Total 886 (delta 198), reused 385 (delta 153), pack-reused 440          
Receiving objects: 100% (886/886), 155.49 KiB | 6.22 MiB/s, done.               
Resolving deltas: 100% (457/457), done.                                         
error: invalid path 'tests/_api_checker_1.10/test_api/Test1DemoApi/get__master_list__{'a': 1}.response.json'
```

**Comment by saxix** [comment]
this is definitely a bug introdiced in https://github.com/saxix/drf-api-checker/commit/d666606c4a75fd6d8bea7e21200cb3ac6c211822#diff-59bc3b97ac2e7d81a0d4500a71af3211

**Comment by saxix** [comment]
@dtroberts can you check last commit ?

**Comment by dtroberts** [comment]
Yep, it's working fine now using the develop branch. Thanks!

**Comment by saxix** [comment]
fixed in [0.10](https://github.com/saxix/drf-api-checker/releases/tag/0.10)
